analyze patest1;
ERROR:  42P01: relation "patest1" does not exist
